core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolverTest.java

```diff
@@ -86,7 +86,11 @@ class AnnotatedValueResolverTest {
     static final Set<String> existingHttpParameters = ImmutableSet.of("param1",
                                                                       "enum1",
                                                                       "sensitive");
-
+    // These parameters will be present without the values. e.g., ?emptyParam1=&emptyParam2=&...
+    static final Set<String> existingWithoutValueParameters = ImmutableSet.of("emptyParam1",
```

Comment:
Should we also add an e2e test in which a client sends a query parameter with a value or an empty value? `EmptyParameterWithDefaultAnnotationTest` could be added for that.

Reply:
Sure. Why not.